src/Client/AzureManaged/DurableTaskSchedulerClientOptions.cs

```diff
@@ -96,6 +96,9 @@ this.Credential is not null
             async (context, metadata) =>
             {
                 metadata.Add("taskhub", taskHubName);
+
+                // Add user agent header with durabletask-dotnet and DLL version from util
+                metadata.Add("user-agent", $"{DurableTaskUserAgentUtil.GetUserAgent()}");
```

Comment:
This covers the client APIs, but shouldn't we also do this for the worker API calls?

Reply:
done